requirements/edx/testing.txt

```diff
@@ -19,3 +19,4 @@ pytest-cov==2.5.1
 pytest-django==3.1.2
 pytest-randomly==1.2.1
 pytest-xdist==1.18.1
+tox==2.8.2
```

Comment:
Do you want to add `tox-battery` now also to recognize changes in requirements file content?

Reply:
Yes - I do! And I will.